.github/workflows/build.yml

```diff
@@ -1,6 +1,10 @@
 name: build
```

Comment:
I wonder if we should merge `build.yml` and `build-and-publish.yml` so that we have only one workflow file?
If we add that `if` statement to the release step, it should never trigger on PR branches.

Reply:
Yes, with that condition it would be much clearer with one single file.